idp-private-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
I don't see a reference, can we remove both checked in certs?

Reply:
the IDP server uses this certificate to generate idp's metadata, we can acutally generate these certificates on the fly, I have just hardcoded for now